src/main/java/com/google/cloud/tools/appengine/api/devserver/DefaultRunConfiguration.java

```diff
@@ -48,6 +49,7 @@
   private String defaultGcsBucketName;
   private Boolean clearDatastore;
   private File datastorePath;
+  private Map<String, String> environment;
```

Comment:
@loosebazooka what are your thoughts, for devappserver2, on what env vars should be set on the process, or passed in as args to --env_var?

We currently set a couple vars on the process itself (JAVA_HOME, CLOUDSDK_CORE_DISABLE_PROMPT). Do you think it makes sense to set all client supplied env vars as --env_var args instead? I'm not sure if there is a difference (and if not, why not just set them all on the process).